pkg/secrets/read_secret.go

```diff
@@ -0,0 +1,130 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I'm sorry but this has nothing to do in the `secrets` package. This should be an external binary that we can include in the container or a command in the cluster-agent (if we really want it in the agent).

The design of `secrets` was to not provide any specific backend for secrets (k8s, docker, vault, aws-kms ...) but just a tooling for people to plug their secret management tool in the agent.

Reply:
@hush-hush thank you, point taken. I can move to a different package for sure, my hope was to reuse this in dd-agent at some point in the near future. I could move this under `cmd/secrets` to resolve your other point about the dependency on `spf13/cobra`. Not having an extra binary seems to be a plus since it makes it trivial to use in the image.

Reply:
What about just reading the file in bash ? This seems pretty trivial task to do and bash would be a good answer to the issue, no ?

Reply:
We'd need to script the json marshal/unmarshal if we were to use bash, plus having this in go has the advantage of being testable.